config/config.go

```diff
@@ -33,6 +33,12 @@ type QueryTweak struct {
 	AlignTimestampsToStep  bool              `yaml:"align_timestamps_to_step" json:"alignTimestampsToStep,omitempty"`
 	DropResultLabels       []model.LabelName `yaml:"drop_result_labels" json:"dropResultLabels,omitempty"`
 	IgnoreFirstStep        bool              `yaml:"ignore_first_step" json:"ignoreFirstStep,omitempty"`
+	FloatCompare           *FloatCompare     `yaml:"float_compare" json:"floatCompare,omitempty"`
+}
```

Comment:
All the other options are "<verb><noun>", descriptive of the tweak they apply. Maybe `AdjustValueDeltaTolerance` here (also because it's specifically about sample values)?

Reply:
Or maybe just `AdjustValueTolerance` is shorter but as clear.

Reply:
I like "float_compare" -> "adjust_value_tolerance" but how do you feel about the names "fraction" and "margin" underneath?

Reply:
I'm happy with those, they seem pretty descriptive and understandable to me, while matching the argument names that they are used as :)